src/apps/defiedge/common/defiedge.strategy.definitions-resolver.ts

```diff
@@ -23,4 +23,17 @@ export class DefiEdgeStrategyDefinitionsResolver {
     const { data } = await axios.get<Strategy[]>(endpoint);
     return data;
   }
+
+  @Cache({
+    key: network => `studio:defiedge:${network}:farming`,
+    ttl: 5 * 60, // 5 minutes
+  })
+  async getFarmingStrategies(network: Network) {
+    const networkParam = network === Network.ETHEREUM_MAINNET ? 'mainnet' : network;
+    const endpoint = `${DEFIEDGE_BASE_URL}/${networkParam}/farming-strategies`;
```

Comment:
just a head'up: 
`https://api.defiedge.io/mainnet/farming-strategies` doesn't return anything

Reply:
Yea, its fine as currently `farming-strategies` are only available on `optimism`. But Thanks 👍 for pointing out.